src/libpanic_abort/lib.rs

```diff
@@ -117,16 +111,16 @@ pub mod personalities {
     //
     // Note that we don't execute landing pads, so this is never called, so it's
     // body is empty.
-    #[no_mangle]
-    #[cfg(all(target_os = "windows", target_env = "gnu"))]
+    #[rustc_std_internal_symbol]
+    #[cfg(all(bootstrap, target_os = "windows", target_env = "gnu"))]
     pub extern "C" fn rust_eh_unwind_resume() {}
 
```

Comment:
This additional requirement on `#cfg(bootstrap)]` looks as though it may be responsible for https://github.com/rust-lang/rust/issues/79609 ... not sure I understand why it was required, however ?

Reply:
The compiler used to emit references to `rust_eh_unwind_resume` from codegen. But it doesn't do that any more and `rust_eh_unwind_resume` doesn't exist any more.